src/nft/hooks/usePermit2Approval.ts

```diff
@@ -7,16 +7,23 @@ import usePermit2Allowance, { AllowanceState } from 'hooks/usePermit2Allowance'
 import { useCallback, useMemo, useState } from 'react'
 import invariant from 'tiny-invariant'
 
+const NFT_UNIVERSAL_ROUTER_MAINNET_ADDRESS = '0x4c60051384bd2d3c01bfc845cf5f4b44bcbe9de5'
```

Comment:
If we updated the UR sdk to the latest version would it have the correct address?

Reply:
discussed in #eng-web we decided this was the best solution

Reply:
can you add a TODO to update the the sdk and import address from there?